janus_core/single_point.py

```diff
@@ -13,13 +14,12 @@
 class SinglePoint:
     """Perpare and perform single point calculations."""
 
-    #  pylint: disable=dangerous-default-value
     def __init__(
         self,
         system: str,
         architecture: str = "mace_mp",
         device: str = "cpu",
-        read_kwargs: dict = {},
+        read_kwargs: dict[str, Any] | None = None,
```

Comment:
I do think we want consistency in type hints. Either all with `__future__` and `|` or all with `Union`. But that can happily be another PR

Reply:
I've dropped `__future__ ` for now, so I think it's all consistent, but happy to revisit later if we decide it was better before (and get get ASE to play along).